tests/js/helpers/helpers-tests.js

```diff
@@ -0,0 +1,52 @@
+/*
+
+  We provide some "helper" grades to assist other people in test routers and other components that run in conjunction
+  with an instance of `gpii.express`.  These tests specifically exercise those test components.
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("./includes.js");
+
+var path    = require("path");
+var viewDir = path.resolve(__dirname, "./views");
```

Comment:
Having gone through this with `gpii-test-browser` already, this was trivial to take care of now.  Done, throughout.